pandas/core/indexes/datetimes.py

```diff
@@ -1964,18 +1966,32 @@ def tz_localize(self, tz, ambiguous='raise', errors='raise'):
 
             .. versionadded:: 0.19.0
 
-        infer_dst : boolean, default False
-            .. deprecated:: 0.15.0
-               Attempt to infer fall dst-transition hours based on order
```

Comment:
You can leave this one (although it will give an error in the validation script, but you can ignore that)

Reply:
so will I add this again ?

Reply:
@jorisvandenbossche why adding this back? looks leftover

Reply:
@jreback @jorisvandenbossche I add this again or not ???????

Reply:
@jreback no, the deprecation is still there (the fact that the deprecation is still there is certainly a left-over, but let's remove that (both the actual deprecation and the docs) in a separate PR)

@HimanshuAwasthi95 Yes, please add back